app/models/claim/base_claim.rb

```diff
@@ -609,7 +609,9 @@ def reset_transfer_court_details
       self.transfer_case_number = nil
     end
 
-    def cleaner; end
+    def cleaner
+      Cleaners::BaseClaimCleaner.new(self)
```

Comment:
🤔  not sure we need this on the `base_claim`. how many claim types actually use cleaners?

Reply:
The main reason for this is so that the `it_behaves_like 'a base claim'` works for all the claim types tests without having to conditionally skip the relevant part.

As a null object, perhaps it would be better to have a new class `Cleaners::NullClaimCleaner` in addition to (and based on) the `BaseClaimCleaner`, to make it clearer what the purpose of this is.

Incidentally, this means that the save navigation operator is not required on line 129, but I haven't changed that (yet).

Reply:
**nit** I do like null objects and it would be clearer

Reply:
are you going to apply the nullobject pattern here?